src/components/HighlightedText/readme.md

```diff
@@ -0,0 +1,85 @@
+##### HighlightedText base
```

Comment:
@amontalvof let's do the new examples in this way:

Each example we should have a title and a description
It should look like the design https://zpl.io/29LEldY